src/react-apps/applications/ux-editor/src/actions/formDesignerActions/actions/updateContainerList/index.ts

```diff
@@ -0,0 +1,108 @@
+import { Action } from 'redux';
+import * as ActionTypes from '../../formDesignerActionTypes';
+
+export interface IUpdateActiveListAction extends Action {
+  listItem: any;
+  containerList: any;
+}
+export interface IUpdateActiveListActionFulfilled extends Action {
+  containerList: any;
+}
+export interface IUpdateActiveListActionRejected extends Action {
+  error: Error;
+}
+
+export interface IUpdateActiveListOrderAction extends Action {
+  containerList: any;
+  orderList: any[];
+}
+export interface IUpdateActiveListOrderActionFulfilled extends Action {
+  containerList: any;
+}
+export interface IUpdateActiveListOrderActionRejected extends Action {
+  error: Error;
+}
+
+export interface IDeleteActiveListAction extends Action {
+}
+
+export interface IDeleteActiveListActionFulfilled extends Action {
```

Comment:
Is it necessary to extend Action if you are not really extending it? Just make it an action.

Reply:
Fixed.